shell/browser/atom_browser_client.cc

```diff
@@ -574,16 +579,6 @@ void AtomBrowserClient::AdjustUtilityServiceProcessCommandLine(
   if (identity.name() == audio::mojom::kServiceName)
     command_line->AppendSwitch(::switches::kMessageLoopTypeUi);
 #endif
-  if (identity.name() == content::mojom::kNetworkServiceName) {
```

Comment:
does this function not get called any more? how come we needed to move this code?

Reply:
Yeah this is only called when the service has identity value associated to it https://cs.chromium.org/chromium/src/content/browser/utility_process_host.cc?l=461 which is no longer the case for network service. I couldn't find a better way to just identify this service, hence attaching the switches to all utility processes. Open to other solutions.

Reply:
nope, seems fine to me.